### PR TITLE
COL-736 Front-end prototype for activity timeline

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-736

This PR gets front-end wiring in place, with no pretensions of conforming to any particular design.

### CODE REVIEW

File: public/app/dashboard/splashController.js [hunk 1]
```diff
@@ -27,15 +27,43 @@
...
+          "Added an asset to a whiteboard": activities.whiteboard_add_asset,
+          "Exported a whiteboard": activities.export_whiteboard
+        };
+
+        $scope.userAssetActivity = {
+          "Viewed my assets": activities.get_view_asset,
+          "Liked my assets": activities.get_like,
+          "Commented on my assets": activities.get_asset_comment,
+          "Replied to my comments": activities.get_asset_comment_reply,
+          "Added my assets to a whiteboard": activities.get_whiteboard_add_asset
```
Comment (paulkerschen) [inline public/app/dashboard/splashController.js]: Note that `get_view_asset` and `get_whiteboard_add_asset` reciprocal activities don't yet exist and will need to be created server-side.

File: public/app/dashboard/splashController.js [hunk 2]
```diff
@@ -27,15 +27,43 @@
 
   'use strict';
 
-  angular.module('collabosphere').controller('SplashController', function(deepLinkId, me, userFactory, $scope) {
+  angular.module('collabosphere').controller('SplashController', function(deepLinkId, me, dashboardFactory, userFactory, $scope) {
```
Comment (johncrossman) [inline public/app/dashboard/splashController.js]: @paulkerschen we prefer alphabetized function args, yes? I.e., `dashboardFactory` first.

**Comment by johncrossman** [comment]
Looks good to me, @paulkerschen  One question posted.

**Comment by paulkerschen** [comment]
Thanks, @johncrossman, args ungarbled.

**Pull request merged**